docs/guides/bulk/upload.md

```diff
@@ -0,0 +1,26 @@
+---
+heading: Bulk
+seo: Bulk | Bulk Download | How to Use Bulk Upload
+title: Upload
+description: How to use Bulk Upload
+layout: sidebarleft
+apis: API Docs
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 3
+---
+
+# Bulk Upload
+
+Unlike Bulk Downloads, using the bulk upload requires only a single call. The `POST /bulk/{objectName}` uploads a file which will then be processed by the bulk job.
```

Comment:
Unlike ~~Bulk Downloads~~bulk downloads, using the bulk upload requires only a single call. The `POST /bulk/{objectName}` uploads a file which will then be processed by the bulk job.